api/tokens_test.go

```diff
@@ -139,12 +141,109 @@ func TestHandleUnexpectedError(t *testing.T) {
 		So(resp.Body.String(), ShouldResemble, errorResponseBodyExample)
 	})
 }
+func TestCognitoResponseBuild(t *testing.T) {
+	Convey("build Cognito Request, an authParams and Config is processed and Cognito Request is built", t, func() {
+
+		authParams := AuthParams{
+			Email:    "email.email@ons.gov.uk",
+			Password: "password",
+		}
+
+		clientId := "awsclientid"
+		clientSecret := "awsSectret"
+		clientAuthFlow := "authflow"
+
+		response := buildCognitoRequest(authParams, clientId, clientSecret, clientAuthFlow)
+
+		So(*response.AuthParameters["USERNAME"], ShouldEqual, authParams.Email)
+		So(*response.AuthParameters["PASSWORD"], ShouldEqual, authParams.Password)
+		So(*response.AuthParameters["SECRET_HASH"], ShouldNotBeEmpty)
+		So(*response.AuthFlow, ShouldResemble, "authflow")
+		So(*response.ClientId, ShouldResemble, "awsclientid")
+	})
+}
+
+func TestCognitoRequestHeaderBuild(t *testing.T) {
```

Comment:
Similar to previous test, should this be TestCognitoResponseHeaderBuild?

Reply:
This has been updated to the suggestion